com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -261,6 +264,27 @@ public bool AutomaticSteppingEnabled
             }
         }
 
+        /// <summary>
+        /// Helper class to receive the JSON package information.
+        /// </summary>
+        [Serializable]
+        struct PackageVersionHelper
+        {
+            #pragma warning disable 0649
+            public string version;
+            #pragma warning restore 0649
+        }
+
+        static string ReadPackageVersion()
+        {
+            // TODO we can potentially replace this in 2019.2 and later with
+            //   UnityEditor.PackageManager.PackageInfo.FindForAssembly(typeof(Agent).Assembly)
```

Comment:
```charp
#if UNITY_2019_2
...
#else
...
#endif
```

Reply:
Disagree, then there are two implementations we have to support, and it's harder to make sure they give the same results.

Reply:
I gave up on this approach, and went back to the hardcoded string + test to make sure it stays consistent.